Firestore/core/src/api/pipeline.cc

```diff
@@ -40,6 +40,20 @@ void Pipeline::execute(util::StatusOrCallback<PipelineSnapshot> callback) {
   this->firestore_->RunPipeline(*this, std::move(callback));
 }
 
+google_firestore_v1_Value Pipeline::to_proto() const {
```

Comment:
Where is this used?

Serializer::EncodePipeline does the same thing. If you want to move it here, we should update the logic there.